### ISSUE TITLE
Crash when opening file with LFO/Op. sequence

### ISSUE THREAD

**Issue by 9001** [rerrahkr/BambooTracker#4]
Regarding the following file: https://ocv.me/tmp/1.btm

When opening this file, BambooTracker crashes. Here is the output:
```
Qt has caught an exception thrown from an event handler. Throwing
exceptions from an event handler is not supported in Qt.
You must not let any exception whatsoever propagate through Qt code.
If that is not possible, in Qt 5 you must at least reimplement
QCoreApplication::notify() and catch all exceptions there.

Segmentation fault
```

The error is somewhere in `FileIO::loadInstrumentPropertySectionInModule`, inside `case 0x00: // FM envelope` when `i == 4`.

Please do not try to rescue data from this file. I am only uploading this file in case you need sample data to help reproduce this crash :-)

I was playing around with all of the operator parameters, and may have created an instrument which is nonsensical/illegal. One of the things I recall doing was to create an `LFO/Operator sequence` for instrument `00`. Parameters: `☐ LFO`, `Operator: AL`, `☑ Sequence`, and the sequence itself was `[2 4 6]` (Loop). When creating a blank song with this configuration, a similar crash is reproduced.

**Comment by rerrahkr** [comment]
Hi, thank you for your report.
I will check for the bug in my environment.

**Comment by OPNA2608** [comment]
It appears this was fixed some time between v0.1.0 and upstream.

v0.1.0: opening the file crashes the application.
upstream: file loads fine. 2 instruments, some notes. although there is no `Sequence` set in instrument `00`.

**Comment by rerrahkr** [comment]
This bug was fixed in [v0.1.1](https://github.com/rerrahkr/BambooTracker/releases/tag/v0.1.1). So I close this issue.
